### ISSUE TITLE
Show min and max amounts in help

### ISSUE THREAD

**Issue by justinpobrien** [lightninglabs/loop#342]
**Issue**
It's not immediately obvious what minimums and maximums are. This could be made more apparent.

**Actual Behavior**
When running `loop out -h` the help text currently shows:
```
USAGE:
   loop out [command options] amt [addr]

OPTIONS:
...
   --amt value                   the amount in satoshis to loop out (default: 0)
...
```

**Expected Behavior**
Would expect to see the min and max visible in help text alongside the default

```
OPTIONS:
...
   --amt value                   the amount in satoshis to loop out (default: 0, min: 250k, max: 60m)
...
```

**Comment by anddddrew** [comment]
> There is the `loop terms` command that will show the minimum and maximum amount the server defines (that will make an RPC request to the server). It's not a common pattern to invoke an RPC just to show the help text of a command (because invoking that RPC might require you to set certain other flags to specify the server/network). So instead maybe we can just add a sentence to the description to check `loop terms` for the minimum and maximum amounts?

Ok.. that sounds good to me

**Comment by hieblmi** [comment]
Fixed by #629
